actions/tics/src/tics/helpers/summary/markdownGenerator.js

```diff
@@ -0,0 +1,63 @@
+const generateLinkMarkdown = (text, link) => {
+    return Boolean(text) && Boolean(link) ? `[${text}](${link})` : '';
+}
+
+const generateStatusMarkdown = (truthCondition, text) => {
```

Comment:
Why not call 'truthCondition' -> passed.

And turn switch into if statement. Return directly rather than assigning to status variable.